docs/source/markdown/podman-auto-update.1.md.in

```diff
@@ -82,73 +67,41 @@ For a container to send the READY message via SDNOTIFY it must be created with t
-registry.fedoraproject.org/fedora:latest   pending
-
-### Autoupdate the services
-$ podman auto-update
-UNIT                    CONTAINER            IMAGE                                     POLICY      UPDATED
-container-test.service  08fd34e533fd (test)  registry.fedoraproject.org/fedora:latest  registry    false
+$ cat ~/.config/containers/systemd/sleep.container
+[Container]                                   
+Image=registry.fedoraproject.org/fedora:latest
+Exec=sleep infinity                           
+AutoUpdate=registry                           
 ```
 
-Autoupdate with local policy
-
+Generate a systemd service from the Quadlet file by reloading the systemd user daemon:
 ```
-### Start a container
-$ podman run --label "io.containers.autoupdate=local" \
-    -d busybox:latest top
-be0889fd06f252a2e5141b37072c6bada68563026cb2b2649f53394d87ccc338
-
-### Generate a systemd unit for this container
-$ podman generate systemd --new --files be0889fd06f252a2e5141b37072c6bada68563026cb2b2649f53394d87ccc338
-/home/user/container-be0889fd06f252a2e5141b37072c6bada68563026cb2b2649f53394d87ccc338.service
-
-### Load the new systemd unit and start it
-$ mv ./container-be0889fd06f252a2e5141b37072c6bada68563026cb2b2649f53394d87ccc338.service ~/.config/systemd/user
 $ systemctl --user daemon-reload
+```
 
-### If the previously created containers or pods are using shared resources, such as ports, make sure to remove them before starting the generated systemd units.
-$ podman stop be0889fd06f252a2e5141b37072c6bada68563026cb2b2649f53394d87ccc338
-$ podman rm be0889fd06f252a2e5141b37072c6bada68563026cb2b2649f53394d87ccc338
-
-$ systemctl --user start container-be0889fd06f252a2e5141b37072c6bada68563026cb2b2649f53394d87ccc338.service
-
-### Get the name of the container
+Start the systemd service and make sure the container is running
+```
+$ systemctl --user start sleep.service
 $ podman ps
-CONTAINER ID  IMAGE                             COMMAND  CREATED        STATUS        PORTS   NAMES
-01f5c8113e84  docker.io/library/busybox:latest  top      2 seconds ago  Up 3 seconds          inspiring_galileo
+CONTAINER ID  IMAGE                                     COMMAND         CREATED        STATUS        PORTS       NAMES
+f8e4759798d4  registry.fedoraproject.org/fedora:latest  sleep infinity  2 seconds ago  Up 2 seconds              systemd-sleep
+```
 
-### Modify the image
-$ podman commit --change CMD=/bin/bash inspiring_galileo busybox:latest
+Check if a new image is available via `--dry-run`:
+```
+$ podman auto-update --dry-run --format "{{.Image}} {{.Updated}}"
+registry.fedoraproject.org/fedora:latest   pending
+```
 
-### Auto-update the container
+Update the service:
+```
 $ podman auto-update
-[...]
+UNIT                    CONTAINER                     IMAGE                                     POLICY      UPDATED
+container-test.service  f8e4759798d4 (systemd-sleep)  registry.fedoraproject.org/fedora:latest  registry    true
```

Comment:
The unit name should be updated to sleep.service.

Reply:
Gosh, sorry.  Didn't apply it to the commit -,- Thanks for double checking.